pipeline/daily/stages.groovy

```diff
@@ -64,7 +66,17 @@ python host_os.py    \
            --updater-email $params.GITHUB_BOT_EMAIL \
            --push-repo-url $VERSIONS_PUSH_REPO_URL \
            --push-repo-branch $COMMIT_BRANCH \
-"""
+""", returnStatus: true)
+
+    Integer SUCCESS_EXIT_CODE = 0
+    Integer NO_UPDATES_EXIT_CODE = 56
+    if (exitCode == SUCCESS_EXIT_CODE) {
+      updateSuccessful = true
+    } else if (exitCode == NO_UPDATES_EXIT_CODE) {
+      updateSuccessful = false
+    } else {
+      error('Packages update failed')
+    }
```

Comment:
The initial value of updateSuccessful is false, I think there is no need to set it again. What about the following?

```groovy
if (exitCode == SUCCESS_EXIT_CODE) {
  updateSuccessful = true
}

if (!updateSuccessful) {
  error('Packages update failed')
}
```